README.md

```diff
@@ -1,6 +1,8 @@
 # SOEN341
 SOEN 341 Project
 
+Lea Lakkis 40125381
```

Comment:
Should be moved to the `Team` section of the README - section was just merged to the `main` branch. Replace student ID with your GitHub user as a link (follow the same format as mine).